src/map.cpp

```diff
@@ -62,16 +63,16 @@ Map::Map(const std::string &name, const SizedType &type, const MapKey &key, int
   mapfd_ = create_map(map_type_, name.c_str(), key_size, value_size, max_entries, flags);
   if (mapfd_ < 0)
   {
-    std::cerr << "Error creating map: '" << name_ << "': " << strerror(errno)
-              << std::endl;
+    LOG(ERROR) << "fail to create map: '" << name_ << "': " << strerror(errno);
   }
 }
 
 Map::Map(const SizedType &type) {
 #ifdef DEBUG
   // TODO (mmarchini): replace with DCHECK
   if (!type.IsStack()) {
-    std::cerr << "Map::Map(SizedType) constructor should be called only with stack types" << std::endl;
+    LOG(ERROR) << "Map::Map(SizedType) constructor should be called only with "
+                  "stack types";
     abort();
```

Comment:
Would a `fatal` log level make sense? It would remove a lot of `abort()` calls

Reply:
Yeah, it sounds reasonable.